templates/ocha-search-block.html.twig

```diff
@@ -0,0 +1,23 @@
+<div class="cd-search">
+  {{ attach_library('ocha_search/google-cse') }}
```

Comment:
This isn't needed for the search block, only the search results page.

Reply:
I'll remove it